lib/Ohmbrewer_Temperature_Sensor.h

```diff
@@ -178,6 +192,11 @@ namespace Ohmbrewer {
              * Digital Pin for the temperature bus one wire protocol
              */
             int _busPin;
+
+            /**
+             * Unique ID for the temperature probe [8] char array ID code
+             */
+            char* _probeId;
```

Comment:
This is useful gogin forward for when we support multiple sensor types and the sensor may be reading on a different pin for a RTD sensor  which dosnt need an ID but will have a bus pin. ....etc.